app/src/main/res/values/strings.xml

```diff
@@ -362,7 +362,7 @@
     <!-- Title for the list of tabs -->
     <string name="tab_header_label">Open tabs</string>
     <!-- Title for the list of tabs in the current private session -->
-    <string name="tabs_header_private_title">Private session</string>
+    <string name="tabs_header_private_title">Private tabs</string>
```

Comment:
I think due to our l10n process we need a new string for this and can't just update the old string